### ISSUE TITLE
Remove old STFC clusters

### ISSUE THREAD

**Issue by mr-field** [UserOfficeProject/stfc-user-office-project#296]
The dev and prod versions of all proposal apps have now been moved to new Kubernetes clusters (https://github.com/UserOfficeProject/stfc-user-office-project/issues/210). However, the old ones are still running on our STFC Cloud project and take up some of our quota.

We should remove the old dev and prod clusters (`test-cluster-4` and `Proposal-Prod-Cluster`), using the [instructions](https://github.com/isisbusapps/ISISBusApps/wiki/Magnum#deleting-clusters) on the wiki:
- [x] Ask STFC Cloud to release the clusters' floating IP
- [ ] Delete the clusters

**Comment by mr-field** [comment]
I've asked STFC Cloud support to remove the cluster IP addresses. The ID of the request is STFCCLOUD-610.

**Comment by mr-field** [comment]
STFC Cloud says they have now removed the floating IPs.

**Comment by mr-field** [comment]
Th old dev and prod clusters have now been removed.

A few odd things that happened while removing them:
- The prod cluster kept re-requesting a new floating IP for its ingress controller after Cloud support removed it. The solution was to delete the cluster's instances and load balancers before asking Cloud support to remove the IP. In the future, we can probably sort this out more cleanly by removing the ingress controller before trying to remove its floating IPs (I couldn't do it this time as the API IP had already been removed)
- When deleting all clusters, Magnum stopped with an error when trying to delete their load balancers. I had to remove them manually, at which point deleting the clusters worked
